cernopendata/modules/fixtures/data/records/delphi-documentation-manuals.json

```diff
@@ -0,0 +1,205 @@
+[
+  {
+    "abstract": {
+      "description": "Detailed description of the DELPHI conditions database contents, and the tools coming with it."
+    },
+    "accelerator": "CERN-LEP",
+      "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "27050"
+    },
+    "date_created": [
+      "1993"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 3
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:18d0bebc",
+        "size": 404470,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/cargo600.pdf"
+      },
+      {
+        "checksum": "adler32:9ef43714",
+        "size": 229767,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/ddapp600.pdf"
+      },
+      {
+        "checksum": "adler32:63ea0076",
+        "size": 557435,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/dplot.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "27001",
+    "title": "DELPHI database documentation",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "DELPHI manuals"
+      ]
+    }
+  },
+  {
+    "abstract": {
+      "description": "DELPHI user analysis framework documentation, Skelana manuals."
+    },
+    "accelerator": "CERN-LEP",
+    "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "27050"
+    },
+    "date_created": [
+      "1999"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 2
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:9a26470d",
+        "size": 174717,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/skelana/99_175_prog_239.pdf"
+      },
+      {
+        "checksum": "adler32:4bac9f7a",
+        "size": 264402,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/skelana/skelana.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "27002",
+    "title": "DELPHI skeleton analysis framework documentation, Skelana.",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "DELPHI manuals"
+      ]
+    }
+  },
+  {
+    "abstract": {
+      "description": "The DELPHI event display manual."
```

Comment:
Cosmetics: the abstract is the same as the title:

![delgra](https://github.com/user-attachments/assets/c7a7f53e-a461-45d6-a787-91f01ce8d0dd)

Perhaps there is something more to say for better readability?

Reply:
Done.